content/300-guides/050-database/800-production-troubleshooting.mdx

```diff
@@ -174,7 +174,7 @@ In this case, you need to fix the non-unique data and then go ahead with the res
 
     ```
 
-    This will create a SQL script file containing all changes necessary to take your production environment from its current failed state to the target state defined in your `prisma.schema`file.
+    This will create a SQL script file containing all changes necessary to take your production environment from its current failed state to the target state defined in your `schema.prisma`file.
```

Comment:
```suggestion
    This will create a SQL script file containing all changes necessary to take your production environment from its current failed state to the target state defined in your `schema.prisma` file.
```